From b8825cd2d4242b13d7ca598211c513c7b1308d84 Mon Sep 17 00:00:00 2001
From: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
Date: Fri, 10 Dec 2021 15:34:19 +0300
Subject: [PATCH 085/120] crtools: remove excess always true condition

Several lines above if (optind >= argc) we go to usage label and fail,
thus we don't need to check (optind < argc) here as it is always true.

Signed-off-by: Pavel Tikhomirov <ptikhomirov@virtuozzo.com>
---
 criu/crtools.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/criu/crtools.c b/criu/crtools.c
index da47bd684..3adfb190f 100644
--- a/criu/crtools.c
+++ b/criu/crtools.c
@@ -124,7 +124,7 @@ int main(int argc, char *argv[], char *envp[])
 		return 1;
 	}
 
-	if (optind < argc && !strcmp(argv[optind], "swrk")) {
+	if (!strcmp(argv[optind], "swrk")) {
 		if (argc != optind + 2) {
 			fprintf(stderr, "Usage: criu swrk <fd>\n");
 			return 1;
-- 
2.34.1

